src/main/java/dev/jbang/cli/CommaSeparatedConverter.java

```diff
@@ -0,0 +1,13 @@
+package dev.jbang.cli;
+
+import java.util.Arrays;
+import java.util.List;
+
+import picocli.CommandLine.ITypeConverter;
+
+public class CommaSeparatedConverter implements ITypeConverter<List<String>> {
+	@Override
+	public List<String> convert(final String input) throws Exception {
+		return Arrays.asList(input.split(","));
```

Comment:
we technically also allow space(" ") and semicolon(";") in //DEPS but I reckon that is not that useful on command line. We can add it later if we see a need.